scripts/install-multi-user.sh

```diff
@@ -753,6 +753,21 @@ EOF
           install -m 0664 "$SCRATCH/nix.conf" /etc/nix/nix.conf
 }
 
+check_umask() {
+    perms=$(umask -S | sed -e 's/[wx]//g')
+    if [ "u=r,g=r,o=r" != "$perms" ]; then
+        failure <<EOF
+The installation will fail with your current umask $(umask).
+Please set your umask to the system default i.e. umask 002
+EOF
+    fi
+}
+
+preflight_checks() {
+    echo "Running Preflight Checks"
```

Comment:
Why not just call this in `validate_starting_assumptions` instead of making a new "stage"?

Reply:
Sure. Done